web/admin/src/App.js

```diff
@@ -241,7 +241,12 @@ export default class App extends PureComponent {
     if (contentItem[prop] !== value) {
       if (value === undefined) value = null
       if (prop === 'type') {
-        pendingContentRef().child(contentItem.key).set({[prop]: value})
+        if (contentItem[prop]) {
```

Comment:
Can you explain this change?

Reply:
So we were running into an issue of the filters are being reset every time you change the type. This works properly but from a UI perspective it is confusing because choosing type is the second task. So this change enables you an opportunity to set the initial type before it will reset on future changes.